recipe/meta.yaml

```
@@ -1,56 +1,149 @@
 {% set name = "auto-sklearn" %}
 {% set version = "0.15.0" %}
 
-
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
     sha256: 082941a56acfc93a83e221e08ef5d5b2083b1ff58d18be7c27d003a11e4a41d0
-  - url: https://raw.githubusercontent.com/automl/automl_common/main/LICENSE
-    sha256: c71d239df91726fc519c6eb72d318ec65820627232b2f796219e87dcf35d0ab4
-    # automl_common license
+  # use GH release archive just for tests
+  - url: https://github.com/automl/{{ name|lower }}/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 99faff39ec6c5362d5ebf2f1330ce5d6ab2a487ef2eff70f387c8092a8af5559
+    folder: gh_src
+    patches:
+      - patches/0001-use-pytest-fixture-for-logger.patch
+      - patches/0002-use-np_longdouble-instead-of-np_float128.patch  # [arm64]
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  # It requires an old version of scikit-learn which we have only
+  # for py38 and py39.
+  # Skip win because pynisher not available
+  skip: True  # [py<37 or py>39 or win]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - pip
-    - python >=3.7
+    - python
+    - setuptools
+    - wheel
   run:
+    - python
     - configspace >=0.4.21,<0.5
-    - dask >=2021.12
+    - dask-core >=2021.12
     - distributed >=2012.12
```

```suggestion
    - distributed >=2021.12
```
I think it's an upstream typo, see https://github.com/automl/auto-sklearn/commit/cd4fef98620cd4be599b9124f06b12a39c2c3526. Must be `2021.12` because the version of `distributed` is always the same as `dask-core`

---

fixed this,
I also pinged the author about this